### ISSUE TITLE
Please change text of totally unclear warning "xyz.js compiled, but not written."

### ISSUE THREAD

**Issue by COLABORATI** [brunch/brunch#1431]
Using [this](https://github.com/bassjobsen/brunch-foundation6.git) repo I get two warnings: 

```
warn: bower_components/what-input/what-input.js compiled, but not written. Check your javascripts.joinTo config
warn: bower_components/jquery/dist/jquery.js compiled, but not written. Check your javascripts.joinTo config
```

It is absolutely unclear what this is about. Please change the text to something meaningful. 
After some research I found out that it seems to be about the `path.watched` still containing the default paths, but not the additional paths that are defined in that repos `brunch-config.coffee` - but I am absolutely not sure about this. However, the files _are_ written, what makes it even more unclear what this is about.

If it is in fact about undefined `path.watched` then please make this warning say exactly that, e.g.:

```
You have configured paths that are not in path.watched.
You should define that path two times in one config file, even if it looks redundant.
```

Or something similar like this. 

If this warning is about something completely different and I did not understand it because the warning is not understandable, then probably I did not understand. I just have a warning that does not help me and the internet does not help me finding an answer. Yes, I will study the code to find what is going on here, but for now any quick hint will do.

Basically, please throw errors and warnings that can be understood. Thanks! :)
Thank you very much for your attention!

**Comment by paulmillr** [comment]
Sounds reasonable.

**Comment by COLABORATI** [comment]
Ah, forgot to ask: is the above presented interpretation of that message right? Do I need to add the path to ```path.watched``` and therefor have it two times in the config? I looked at the code but I do not understand what it should do.

**Comment by es128** [comment]
No, that interpretation is not correct.

What the message is trying to say is that this file does show up somewhere under the paths in `paths.watched` and has a file extension expected by your compiler plugin(s), so brunch has done work to compile it... BUT it does not match any of your configured `joinTo` definitions, and therefore is not getting concatenated into any of your final output files.
